extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/TenantContextFactory.java

```diff
@@ -199,7 +198,7 @@ private Uni<TenantConfigContext> createTenantContext(OidcTenantConfig oidcTenant
                         LOG.debugf("Default tenant is not configured and will be disabled"
                                 + " because either 'TenantConfigResolver' which will resolve tenant configurations is registered"
                                 + " or named tenants are configured.");
-                        oidcConfig.tenantEnabled = false;
+                        oidcConfig.tenantEnabled = true;
```

Comment:
What is the idea behind changing it to `true` - if it is expected to be disabled ?

Reply:
funny no test failed, this is typo I made when putting back what was there before

Reply:
Thanks for catching it, this was terrible mistake.